src/table/utils/styling-utils.js

```diff
@@ -14,13 +14,13 @@ export const STYLING_DEFAULTS = {
 
 export const SELECTION_STYLING = {
   SELECTED: {
-    color: STYLING_DEFAULTS.WHITE,
+    color: `${STYLING_DEFAULTS.WHITE} !important`,
```

Comment:
To use sx in the withStyling.jsx, I have to add the '!important'

Reply:
This is weird, do other font colors work? we only add them in one place right?

Reply:
https://github.com/qlik-oss/sn-table/pull/435/files#diff-d6ca75cf192bf2d9f1b1c0a33da28c36feea8dea33aaa62eb43727951dd1077cR39 we have font color here as well. that is the reason I need to add the '!important'.

Reply:
I managed to remove the `!important` by moving the color in tableBodyStyle to styling in CellRenderer as shown in https://github.com/qlik-oss/sn-table/pull/435/files#diff-0d7173fd8da1e342bacca6f203bd521b0300c9ab25684eb6817b7e3d4be57a0cR86